<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>120</width>
      <height>125</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rbC">
       <property name="text">
        <string>C++</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbCSharp">
       <property name="text">
        <string>C#</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbJava">
       <property name="text">
        <string>Java</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>98</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="rbRed">
       <property name="text">
        <string>Vermelho</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbGreen">
       <property name="text">
        <string>Verde</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbBlue">
       <property name="text">
        <string>Azul</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>98</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="rbPlane">
       <property name="text">
        <string>Avi√£o</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbCar">
       <property name="text">
        <string>Carro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbShip">
       <property name="text">
        <string>Navio</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>353</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
